pkg/scheduler/apis/config/v1alpha1/defaults.go

```diff
@@ -34,8 +34,8 @@ func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
 
-// SetDefaults_KubeSchedulerConfiguration sets additional defaults
-func SetDefaults_KubeSchedulerConfiguration(obj *kubescedulerconfigv1alpha1.KubeSchedulerConfiguration) {
+// SetDefaultsKubeSchedulerConfiguration sets additional defaults
+func SetDefaultsKubeSchedulerConfiguration(obj *kubescedulerconfigv1alpha1.KubeSchedulerConfiguration) {
```

Comment:
the underscore style used everywhere else (and maybe even required by the generator)

Reply:
sorry, You are right, the underscore style is used everywhere。 I close this。